phoenix-core/src/main/java/org/apache/phoenix/mapreduce/AbstractBulkLoadTool.java

```diff
@@ -226,6 +228,14 @@ private int loadData(Configuration conf, CommandLine cmdLine) throws Exception {
         configureOptions(cmdLine, importColumns, conf);
         String sName = SchemaUtil.normalizeIdentifier(schemaName);
         String tName = SchemaUtil.normalizeIdentifier(tableName);
+
+        // There must be a better solution for this
+
+        String tn = sName == null ? "\"" + tName + "\"" : "\"" + sName + "\".\"" + tName + "\"";
+        ResultSet rsempty = conn.createStatement().executeQuery("SELECT * FROM " + tn);
```

Comment:
Add limit 1 to hopefully reduce the load.

Reply:
Also I'm sure we already have a utility function somewhere to generate the table name from the components.